robusta_krr/core/integrations/kubernetes.py

```diff
@@ -109,28 +115,47 @@ async def __build_obj(
             pods=await self.__list_pods(item),
         )
 
+    async def _list_resources(self, all_namespaces_fn, namespaced_fn):
+        resources = []
+        if self.config.namespaces == "*":
```

Comment:
I think here you accidentally lost the logic where `kube-system` is ignored by default (when scanning with `"*"`)